### ISSUE TITLE
IT Support: I am looking to get some analytics on the usage of my Power BI reports.

### ISSUE THREAD

**Issue by Matthew-M-TPW** [cityofaustin/atd-data-tech#17725]
BOP

## What's the problem?
"I am looking to get some analytics on the usage of my Power BI reports. I need to know who my Admin for Power BI is so I can work with them to enable this type of reporting."

The ticket came from CTM INC0400539

**Comment by Charlie-Henry** [comment]
@susannegov I think we need a CTM power BI admin to take a look at this for us. I've noticed that the dataset which powers the usage reports have not been updated in 9 months or so? When I try to view the semantic model, it says it doesn't exist which makes me think that it's only visible to CTM admins?

<img width="1178" alt="Screenshot 2024-06-07 at 2 04 37 PM" src="https://github.com/cityofaustin/atd-data-tech/assets/30810522/44200a5b-eee4-4541-b8e2-70060fc30cd3">

**Comment by susannegov** [comment]
Yeah, this needs to go to the CTM Power BI admin as the usage reports semantic model are not managed by workspace admins. @Matthew-M-TPW the ServiceNow ticket would need to get rerouted to the Sharepoint Support Team or ESA

**Comment by Matthew-M-TPW** [comment]
Created new CTM ticket since i was unable to open a new one INC0401100

**Comment by susannegov** [comment]
Reached out to Matt on any updates to the ticket. He said it hasn't been worked on, but he submitted a comment today to see if there are any updates.

**Comment by susannegov** [comment]
https://github.com/cityofaustin/atd-data-tech/issues/16739#issuecomment-2251038106
> Checked July 25 and July 26 and it is refresh daily

**Comment by Charlie-Henry** [comment]
I think we're good to close this out @susannegov? Looks like it's working now
